### ISSUE TITLE
Hilight text-entry box upon joining a game (the pre-"I'm Ready"-screen)

### ISSUE THREAD

**Issue by blarum** [wesnoth/wesnoth#3417]
Hilight text-entry box upon joining a game (the pre-"I'm Ready"-screen). Send all keystrokes there.

**Comment by Wedge009** [comment]
I'm not sure what this is asking. I'm assuming it's for the MP Lobby. I think the 'pre-"I'm ready"-screen' is called 'MP Staging'. My guess is that you want the initial focus on this screen to be on the chat box - otherwise I don't really understand what is meant by 'hilight'/highlight.

If this is the case, #2817 might be somewhat related. There may be other more closely related requests but I can't find them right now.

**Comment by jostephd** [comment]
Yes, I think the patch I posted to #2817 fixes this too. It's not _quite_ a duplicate (that issue is about focus loss, this issue is about initial focus) but the patch fixes both of them.

**Comment by Wedge009** [comment]
Ah, I notice that patch isn't live yet. Anything stopping us from adding it now or at least making a PR?

**Comment by jostephd** [comment]
I might be being overly conservative here, but 1.15.2 was only just tagged so I don't push yet changes that haven't had much testing, in case some problem is found in 1.15.2 and we need to re-tag it. #2817 is already assigned to me and labeled "Ready for testing". I could do a PR too if people prefer that way, but one way or the other, I do monitor my assigned issues page.

Speaking of which, asking developers to unassign themselves of issues they aren't actively working on would probably be a good idea. We currently have 89 open issues with assignees and I doubt they're all being worked on.

**Comment by Wedge009** [comment]
Oh, I wasn't saying you don't check your assignments at all. I'm aware we have 1.15.2 tagged, but I didn't think that meant we can't continue development on master until 1.15.2 is actually released, does it?

The assignment question is probably separate to this issue discussion. If you're looking after that patch, then let's wait till it's committed before we continue testing this.

**Comment by jostephd** [comment]
Oh, I was just trying to say that #2817 is on my radar so it won't get lost. Didn't read any implication to the contrary either, I just wrote that too quickly. Sorry for the misunderstanding.

About master, no, master isn't locked for changes, I'm just taking large safety margins.

**Comment by Wedge009** [comment]
#2817 was merged but looks like there's still no initial focus as of 1.15.17+dev (1.16 beta).
